src/graph/unit_graph.cc

```diff
@@ -977,4 +1132,31 @@ HeteroGraphPtr UnitGraph::GetAny() const {
   }
 }
 
+HeteroGraphPtr UnitGraph::GetFormat(SparseFormat format) const {
+  if (format == SparseFormat::CSR)
+    return GetOutCSR();
+  else if (format == SparseFormat::CSC)
+    return GetInCSR();
+  else if (format == SparseFormat::COO)
+    return GetCOO();
+  else if (format == SparseFormat::ANY)
+    return GetAny();
+
+  LOG(FATAL) << "Unsupported format code: " << format;
+  return nullptr;
```

Comment:
One benefit of using `enum class` is that the compiler will warn you about a missing condition when using `switch ... case ...`.